### PR TITLE
Update: font-face3 added back TORO-38098

### PR DESCRIPTION
**Pull request by ejaz-tapestry**

### CODE REVIEW

**Review by aoverstreet-tapestry — APPROVED** [review]

File: src/brands/kate-spade/default/font/face.json [hunk 1]
```diff
@@ -70,6 +70,48 @@
...
+          "stretch": {
+            "value": "normal"
+          },
+          "local": {
+            "value": ["New Baskerville ITC Pro"]
+          },
+          "woff2": {
+            "value": "https://assets.katespade.com/na/fonts/ITCNewBaskervilleRoman.woff2"
+          }
+        }
```
Comment (p-musi) [inline src/brands/kate-spade/default/font/face.json:114]: We don't need to define the font more than once, we can just have the file below reference "2" when grabbing the font value.

File: src/brands/kate-spade/default/font/family.json [hunk 2]
```diff
@@ -13,6 +13,14 @@
         "normal": {
           "value": "{font.face.2.normal.family.value}, Baskerville, Georgia, serif"
         }
+      },
+      "face3": {
+        "normal": {
+          "value": "{font.face.3.normal.family.value}, serif"
```
Comment (p-musi) [inline src/brands/kate-spade/default/font/family.json:19]:
```suggestion
          "value": "{font.face.2.normal.family.value}, serif"
```
By having this reference "2", which is already defined, we don't need to define a "3" in the first file.

File: src/brands/kate-spade/default/font/family.json [hunk 3]
```diff
@@ -13,6 +13,14 @@
...
+        },
+        "light": {
+          "value": "{font.face.3.light.family.value}, serif"
```
Comment (p-musi) [inline src/brands/kate-spade/default/font/family.json:22]:
```suggestion
          "value": "{font.face.2.normal.family.value}, serif"
```
By having this reference "2", which is already defined, we don't need to define a "3" in the first file.

**Review by p-musi — CHANGES_REQUESTED** [review]

File: src/brands/kate-spade/default/font/family.json (hunk 3 above)
Comment (ejaz-tapestry) [reply src/brands/kate-spade/default/font/family.json:22]: changes done

File: src/brands/kate-spade/default/font/face.json (hunk 1 above)
Comment (ejaz-tapestry) [reply src/brands/kate-spade/default/font/face.json:114]: extra code removed

File: src/brands/kate-spade/default/font/family.json (hunk 2 above)
Comment (ejaz-tapestry) [reply src/brands/kate-spade/default/font/family.json:19]: changes done

**Review by p-musi — APPROVED** [review]

**Pull request merged**